### DISCUSSION TITLE
Cannot preview an image within chatroom

### DISCUSSION THREAD

**Discussion by KelvinHUNG11** [advanced-chat/advanced-chat-components#357] [Q&A] [answered]
Hi there, I am a beginner in chatroom development. I am now using the vue-advanced-chat for my development.
It's a really good and user-friendly module for me.
But I faced a difficulty which drags me for days to be resolve, so I try to post it here for help. Your help is appreciated.

Here is the story, I tried to send an image to a chatroom and wanna preview it, however, it failed. Below is the frontend return msg.
The image I sent is : 
https://firebasestorage.googleapis.com/v0/b/vuexfire-a9c43.appspot.com/o/files%2F6R0MijpK6M4AIrwaaCY2%2F5RM2yf2TBCVmAUpaHbCD%2Fimage.png?alt=media&token=141dc7d8-1665-438b-871e-6e8566fabd2c

**** msg successfully sent out and received through vue-socket.io ****
![Screenshot 2022-09-16 at 4 17 21 PM](https://user-images.githubusercontent.com/95609920/190591162-3f1db9de-c8e2-48aa-b3a8-8321db0f0977.png)

**** cannot preview in the chatroom at sender's side ****
![Screenshot 2022-09-16 at 4 21 33 PM](https://user-images.githubusercontent.com/95609920/190591976-1a8c2f23-9585-403e-b5b3-f2cab1a15fda.png)

**** cannot preview in the chatroom at receiver's side ****
<img width="1100" alt="Screenshot 2022-09-16 at 4 23 05 PM" src="https://user-images.githubusercontent.com/95609920/190592317-69d5c0f5-2462-4226-a40e-9351f20f4d18.png">

May I know if there is anyone can help? thanks.

**Comment by antoine92190** [comment, marked as answer]
Why do you pass `audio: true` to the file?
Image files should look like this:
```
{
  name: 'My Replied File',
  size: 67351,
  type: 'png',
  url: 'https://firebasestorage.googleapis.com/...'
}
```

**Reply by KelvinHUNG11** [reply]
thanks for pointing out, I removed it and tried again. However, the same issue is still existed. 
I have set the "media-preview-enabled" = true 

Below is the msg record after removing the redundant attributes. 

****msg successfully sent out and received through vue-socket.io ****
![Screenshot 2022-09-16 at 4 53 25 PM](https://user-images.githubusercontent.com/95609920/190598687-06a57958-6967-4f84-8525-bd56fdcd413d.png)


**** cannot preview in the chatroom at receiver's side ****
![Screenshot 2022-09-16 at 4 55 11 PM](https://user-images.githubusercontent.com/95609920/190598800-e591c7ed-5d2a-489f-b4ee-9ca158c68075.png)

**Reply by antoine92190** [reply]
Your file object looks stringified. Make sure it's an object, and not the string of an object

**Reply by KelvinHUNG11** [reply]
thanks for pointing out!! It works now
